Dockerfile.sriov-network-config-daemon.rhel7

```diff
@@ -1,10 +1,10 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
```

Comment:
we probably want to remove the rhel7 dockerfiles in k8s plumbing repo.

Reply:
fixed